reports/saksham.md

```diff
@@ -0,0 +1,268 @@
+ - 22:
+   - repo: coala-utils
+   - link: https://gitlab.com/coala/coala-utils/commit/18c91bb2824e6bcb09de14af3ae38ec279b8d9e3
+   - details: >
+      use __all__ for exports to prevent failing coala --ci check
+ - 23:
+   - repo: coala-utils
+   - link: https://gitlab.com/coala/coala-utils/commit/25cc5a3178586a1d51d0385eca12a176914a0b8c
+   - details: >
+      Use appveyor template for coala-utils
+ - 24:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/commit/c35a2ed04935fd7665514e6af7828f1bfd931a03
+   - details: >
+      Improve and cleanup script for generating bear requirements and generate cabal requirements
+ - 25:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/commit/26b696969dd38ca1a1ab91163e0154f6c905e252
+   - details: >
+      Move the generate_package script from the coala-bears package to .ci directory
+ - 26:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/commit/efdfbc1cec98db76a5bb2c75fbfa414686be4db8
+   - details: >
+      Use CabalRequirement for HaskellLintBear
+ - 27:
+   - repo: coala
+   - link: https://github.com/coala/coala/commit/4422063fa78e6b0e0c801ea845974367baabdb69
+   - details: >
+      Run coala static code analysis on repo using appveyor
+ - 28:
+   - repo: dependency_management
+   - link: https://gitlab.com/coala/package_manager/commit/75b3c37f7ffd7b6c9f693d131bf4f627fd95373d
+   - details: >
+      Add PlatformRequirement class for system specific package managers
+ - 29:
+   - repo: dependency_management
+   - link: https://gitlab.com/coala/package_manager/commit/a786450e5cdfd3172caf86c60786c0d1ce6ee10f
+   - details: >
+      Return exitcode after running the install command
+ - 30:
+   - repo: dependency_management
+   - link: https://gitlab.com/coala/package_manager/commit/868afe0dcdf6ab0c964f1ce72152fedb132f44cb
+   - details: >
+      Add CabalRequirement for backwards compatibility
+ - 31:
+   - repo: dependency_management
+   - link: https://gitlab.com/coala/package_manager/commit/995b50ba1861296d333442be9e16ba2b1c4a47b6
+   - details: >
+      Run pip as python module
+ - 32:
+   - repo: bear-install-playbook
+   - link: https://gitlab.com/coala/bear-install-playbook/commit/7ba1c3ef9c29dc58a1c5b0109328317f3b0924df
+   - details: >
+      Add playbook for installing individual bears
+ - 33:
+   - repo: bear-install-playbook
+   - link: https://gitlab.com/coala/bear-install-playbook/commit/1b288844aa157ff4e8c80a3179c5d58163a2719f
+   - details: >
+      Test playbook using docker + Travic CI
+ - 34:
+   - repo: bear-install-playbook
+   - link: https://gitlab.com/coala/bear-install-playbook/commit/7dee7591927c0883269f5741102b90bfb93f84ff
+   - details: >
+      Add instructions on using playbook
+ - 35:
+   - repo: bear-install-playbook
+   - link: https://gitlab.com/coala/bear-install-playbook/commit/c82a9737c6852d09f681a7fc98a0f094dd38194b
+   - details: >
+      Run coala static code analysis
```

Comment:
this was an automated change? if so, discard

Reply:
removed all the automated changes :)